Babylon/groups/api/groups/dataset/commands/delete.py

```diff
@@ -0,0 +1,80 @@
+from logging import getLogger
+
+from click import argument
+from click import option
+from click import command
+from click import confirm
+from click import make_pass_decorator
+from click import prompt
+from cosmotech_api.api.dataset_api import DatasetApi
+from cosmotech_api.exceptions import NotFoundException
+from cosmotech_api.exceptions import UnauthorizedException
+
+from ......utils.decorators import allow_dry_run
+from ......utils.decorators import timing_decorator
+from ......utils.decorators import require_deployment_key
+
+logger = getLogger("Babylon")
+
+pass_dataset_api = make_pass_decorator(DatasetApi)
+
+
+@command()
+@allow_dry_run
+@pass_dataset_api
+@timing_decorator
+@require_deployment_key("organization_id", "organization_id")
+@option(
+    "-f",
+    "--force",
+    "force_validation",
+    is_flag=True,
+    help="Don't ask for validation before delete",
+)
+@argument("dataset_id", type=str, required=True)
```

Comment:
Add input file option, see PR #29

Reply:
If dataset id is grabbed from the file then dataset_id is not always needed as an argument